wasm/src/bindings_skia.cpp

```diff
@@ -20,98 +20,110 @@
+                                                      kBottomLeft_GrSurfaceOrigin,
+                                                      kRGBA_8888_SkColorType,
+                                                      nullptr,
+                                                      nullptr)
+            .release();
+    }
+
+    void saveClipRect(float l, float t, float r, float b)
+    {
+        save();
+        std::unique_ptr<rive::RenderPath> rect(jsFactory()->makeEmptyRenderPath());
+        rect->moveTo(l, t);
+        rect->lineTo(r, t);
+        rect->lineTo(r, b);
+        rect->lineTo(l, b);
+        rect->close();
+        clipPath(rect.get());
+    }
+
+    void restoreClipRect() { restore(); }
 };
 
-WebGLSkiaRenderer *makeSkiaRenderer(int width, int height) {
-  GrContextOptions options;
-  sk_sp<GrDirectContext> context = GrDirectContext::MakeGL(nullptr, options);
-  return new WebGLSkiaRenderer(context, width, height);
+WebGLSkiaRenderer* makeSkiaRenderer(int width, int height)
+{
+    GrContextOptions options;
+    sk_sp<GrDirectContext> context = GrDirectContext::MakeGL(nullptr, options);
+    return new WebGLSkiaRenderer(context, width, height);
 }
 
-EMSCRIPTEN_BINDINGS(RiveWASM_Skia) {
-  class_<rive::Renderer>("Renderer")
-      .function("save", &rive::Renderer::save)
-      .function("restore", &rive::Renderer::restore)
-      .function("transform", &rive::Renderer::transform, allow_raw_pointers())
-      .function("drawPath", &rive::Renderer::drawPath, allow_raw_pointers())
-      .function("clipPath", &rive::Renderer::clipPath, allow_raw_pointers())
-      .function("align", &rive::Renderer::align, allow_raw_pointers());
-  class_<WebGLSkiaRenderer, base<rive::Renderer>>("WebGLRenderer")
-      .function("clear", &WebGLSkiaRenderer::clear)
-      .function("flush", &WebGLSkiaRenderer::flush)
-      .function("resize", &WebGLSkiaRenderer::resize)
-      .function("saveClipRect", &WebGLSkiaRenderer::saveClipRect)
-      .function("restoreClipRect", &WebGLSkiaRenderer::restoreClipRect);
-
-  function("makeRenderer", &makeSkiaRenderer, allow_raw_pointers());
+EMSCRIPTEN_BINDINGS(RiveWASM_Skia)
+{
+    class_<rive::Renderer>("Renderer")
+        .function("save", &rive::Renderer::save)
+        .function("restore", &rive::Renderer::restore)
+        .function("transform", &rive::Renderer::transform, allow_raw_pointers())
+        .function("drawPath", &rive::Renderer::drawPath, allow_raw_pointers())
+        .function("clipPath", &rive::Renderer::clipPath, allow_raw_pointers())
+        .function("align",
+                  optional_override([](rive::Renderer& self,
+                                       rive::Fit fit,
+                                       JsAlignment alignment,
+                                       const rive::AABB& frame,
+                                       const rive::AABB& content) {
+                      self.align(fit, convertAlignment(alignment), frame, content);
```

Comment:
We override the default align with one that accepts our enum and converts it to the matching rive::Alignment. This allows us to only expose the enum to JS.